dll/win32/shell32/CShellLink.cpp

```diff
@@ -357,6 +362,12 @@ HRESULT STDMETHODCALLTYPE CShellLink::Save(LPCOLESTR pszFileName, BOOL fRemember
 
         if (SUCCEEDED(hr))
         {
+            if (!bAlreadyExists)
+            {
+                GetFullPathNameW(pszFileName, _countof(szFullPath), szFullPath, NULL);
+                SHChangeNotify(SHCNE_CREATE, SHCNF_PATHW, szFullPath, NULL);
+            }
```

Comment:
Is it needed to send an `SHCNE_UPDATEITEM` in the else case?

Reply:
![yeah](https://user-images.githubusercontent.com/2107452/78870291-d79cc700-7a80-11ea-8025-14af85b02975.png)
Yeah. OK.